docs/TektonConfig.md

```diff
@@ -284,6 +289,52 @@ chain:
   transparency.url: #value
 ```
 
+### Result
+
+Result section allows user to customize the Tekton Result component, Refer to [Result Spec](https://github.com/tektoncd/operator/blob/main/docs/TektonResult.md#spec) section in TektonResult for available options.
+
+Default Result configuration in TektonConfig looks like following if user doesn't specified any configuration options
+
+Example:
+
+```yaml
+result:
+  disabled: false
+  is_external_db: false
+  options: {}
+```
+
+User can customize Result configuration with following options
+
+Example:
+
+```yaml
+result:
+  disabled: false                  # - `disabled` : if the value set as `true`, result component will be disabled (default: `false`)
+  targetNamespace: tekton-pipelines
+  is_external_db: false # If it's true use external database and Result gets it's database from kubernetes secret named `tekton-results-postgres`
```

Comment:
By default, this is set to false, TektonOperator will create  Tekton Results database. If set to true, an external database will be used, and Tekton Results will retrieve its database credentials from the Kubernetes secret named tekton-results-postgres